setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='edx-rest-api-client',
-    version='1.4.0',
+    version='1.4.1',
```

Comment:
I think you want 1.5.0. The changes you're introducing here aren't a bug fix. You're adding functionality in a backwards-compatible manner. See http://semver.org/ for more.